inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution_negative.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "shared_test_classes/single_layer/convolution.hpp"
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/test_constants.hpp"
+#include "../skip_tests_check.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16
+};
+
+/* ============= 2D Convolution ============= */
+const std::vector<std::vector<size_t >> kernels2D = {
+    {1, 3},
+    {7, 1},
+    {3, 3},
+};
+
+const std::vector<std::vector<size_t >> InvalidKernels2D = {
```

Comment:
Could be changed to kernels2DInvalid to align with other names.

Reply:
I will improve with the next PR